plugins/modules/ini_file.py

```diff
@@ -116,6 +116,14 @@
       - Allow option without value and without '=' symbol.
     type: bool
     default: false
+  modify_inactive_option:
+    description:
+      - By default the module replaces a commented line that matches the given option.
+      - Set this option to V(false) to avoid this. This is useful when you want to keep example
```

Comment:
I would make this more explicit by also adding 'commented':
```suggestion
      - Set this option to V(false) to avoid this. This is useful when you want to keep commented example
```